packages/medusa/src/loaders/feature-flags/index.ts

```diff
@@ -0,0 +1,70 @@
+import path from "path"
+import glob from "glob"
+
+import { FlagSettings } from "../../types/feature-flags"
+import { FlagRouter } from "../../utils/flag-router"
+import { Logger } from "../../types/global"
+
+const isTruthy = (val: string | boolean | undefined): boolean => {
+  if (typeof val === "string") {
+    return val.toLowerCase() === "true"
+  }
+  return !!val
+}
+
+export default (
+  configModule: { featureFlags?: Record<string, string | boolean> } = {},
+  logger?: Logger,
+  flagDirectory?: string
+): FlagRouter => {
+  let { featureFlags: projectConfigFlags } = configModule
+  projectConfigFlags = projectConfigFlags || {}
+
+  const flagDir = path.join(flagDirectory || __dirname, "*.js")
+  const supportedFlags = glob.sync(flagDir, {
+    ignore: ["**/index.js"],
+  })
+
+  const flagConfig: Record<string, boolean> = {}
+  for (const flag of supportedFlags) {
+    let flagSettings: FlagSettings
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const importedModule = require(flag)
+    if (importedModule.default) {
```

Comment:
suggestion: We can always try to reduce the statement by doing the following:
```ts
if (!importedModule.default) {
  continue
}
```

Generally speaking it allow us the reduce the indentation accumulation and get something more readable.